lore/settings.py

```diff
@@ -67,6 +67,7 @@ def get_var(name, default):
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'taxonomy.apps.TaxonomyConfig',
```

Comment:
The import in `INSTALLED_APPS` name is usually single value (like `TaxonomyConfig`, and lower-case. So the "app" may be on the wrong path. Also, it appears that `TaxonomyConfig` is a class, and the class name isn't what's added to `INSTALLED_APPS`; the app folder itself (the one usually containing `models.py`) is.